packages/api-clients/open-api/tenantApi.yml

```
@@ -792,7 +792,7 @@ paths:
       tags:
         - tenant
       summary: Deletes a tenant
-      operationId: maintenanceTenantDelete
+      operationId: maintenanceTenantDeleted
```

I guess this was the original expected change
```suggestion
      operationId: maintenanceTenantDelete
```